helper/test/Makefile.am

```diff
@@ -1,11 +1,11 @@
 include $(top_srcdir)/test/Makefile.inc
 
-EXECUTABLES = version \
-	      debug \
-	      chksum \
-	      macros \
-	      parse \
-	      stress
+EXECUTABLES = odph_version \
+	      odph_debug \
+	      odph_chksum \
+	      odph_macros \
+	      odph_parse \
+	      odph_stress
```

Comment:
For uniformity I'd prefix also the tests below to `linux/odph_pthread`, `linux/odph_process`, and `odph_cli`.

Reply:
Thanks @MatiasElo for the review
I will send V2 with suggested changes